app/src/main/java/de/westnordost/streetcomplete/quests/sport/AddSport.java

```diff
@@ -0,0 +1,48 @@
+package de.westnordost.streetcomplete.quests.sport;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddSport extends SimpleOverpassQuestType
+{
+	@Inject public AddSport(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways with leisure=pitch and !sport";
```

Comment:
I did not look for relations as they take much longer time to retrieve against Overpass API and they represent less than 0.5% of all pitches. So pitches relations without sport tag are only a few (for example 44 over all France), I'm not sure it is worth the look-up cost.

Why not check for deprecated tags, but this shouldn't be another quest ?